.travis.yml

```diff
@@ -12,7 +12,7 @@ services:
 - docker
 language: go
 go:
-- 1.12.9
+- 1.13.x
```

Comment:
_n: better to stick to a specific version to avoid flaky builds?

Reply:
reverted